packages/compat/src/v1-addon.ts

```diff
@@ -729,6 +729,12 @@ export default class V1Addon {
   private buildAddonStyles(built: IntermediateBuild) {
     let addonStylesTree = this.addonStylesTree();
     if (addonStylesTree) {
+      let destDir = (addonStylesTree as any).destDir;
+
+      if (typeof destDir === 'string' && destDir.endsWith('__COMPILED_STYLES__')) {
```

Comment:
just a quick work-around to test CI

Reply:
Some some thoughts:

* chat with @rwjblue to verify my understanding of the `__COMPILED_STYLES__` change in ember-cli
* I don't believe we have a built-in relocating system for css, so if this path is the right approach we will need to implement something.
* One possible approach is: 1) if we detect this exact case, apply this exact fix ... otherwise 2) implement a broccoli funnel or plugin which undoes the nesting if needed.